<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>inputNFA</string>
     </attribute>
     <widget class="QPushButton" name="btnGenerate">
      <property name="geometry">
       <rect>
        <x>1230</x>
        <y>30</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtIn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>941</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="CustomView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1291</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Verify</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditAFD">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>701</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVerifyAFND">
      <property name="geometry">
       <rect>
        <x>1210</x>
        <y>30</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>verify</string>
      </property>
     </widget>
     <widget class="CustomView" name="graphicsView2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1281</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRespuestaVerify">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>401</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Clear"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
